src/main/java/com/documaster/rms/noark/ws/noarkentities/MappeBase.java

```diff
@@ -215,6 +219,46 @@ public void setMappetype(Mappetype mappetype) {
 		return link(NOEKKELORD_LINK, noekkelord);
 	}
 
+	public LinkAction<TEntity> addKryssReferanseTilMappe(String... mappeIds) {
+
+		return link(KRYSSREFERANSE_TIL_MAPPE_LINK, mappeIds);
+	}
+
+	public LinkAction<TEntity> addKryssReferanseTilMappe(AbstraktMappe... mappeIds) {
```

Comment:
Parameter `mappeIds` should be changed to just `mappe` (?).

Applies to all methods that accept Noark objects.

Reply:
Fixed.